sourmash/commands.py

```diff
@@ -731,107 +737,130 @@ def multigather(args):
 
     # run gather on all the queries.
     for queryfile in inp_files:
```

Comment:
ok - this part is now much easier with `load_file_as_signatures`! It becomes a double loop, I'm sorry to say, but something like:

```
for queryfile in inp_files:
   for query_sig in sourmash_args.load_file_as_signatures(queryfile, ksize=..., select_moltype=...):
      (proceed as normal)
```